opm/simulators/wells/BlackoilWellModelGeneric.cpp

```diff
@@ -1516,7 +1516,7 @@ updateAndCommunicateGroupData(const int reportStepIdx,
                     break;
                 }
                 default:
-                    assert(false); //programming error
+                    throw std::logic_error("MULTI-phase injector is not supported.");
```

Comment:
You can add well name here,  so that we know something wrong with certain well. 

`injector` can be `injection`.

Reply:
Amended.